### ISSUE TITLE
why PEtal is down ?

### ISSUE THREAD

**Issue by deqee** [kolton/d2bot-with-kolbot#761]
Do you guys have other websit like www.projectetal.com or blizzhackers.cc both are down

**Comment by TheySeeHimRolling** [comment]
In issue #722 I asked the same question and someone said it wasn't down, which clearly it is tho.
So I checked out other forums, but none seemed to have an answer. Sooooo I asked under a kolton yt Video and got the following response: "It's kinda on "hold" for lack of a better term. No idea if it will ever return﻿" 

So this is the best I can give you @the moment.

**Comment by uniquelolram** [comment]
unfortunate, there was alot of great information on those forums.

**Comment by DarkHorseDre** [comment]
Yeah has been down for weeks.  Its a shame as there are guides there for Kolton which are not documented here :(

**Comment by mf022** [comment]
check the discord channels and the https://d2bot.discourse.group.

bh pages can be found on web archive. https://web.archive.org/web/*/www.blizzhackers.cc
( check also this topic https://d2bot.discourse.group/t/bh-on-archive-org/43)

etal pages can be found on https://web.archive.org/web/*/projectetal.com , but the wiki was saved last time on June 2017.

My opinion about etal closing :
project etal was closed by admin(Ethic) with no other notification, than a 15 days counter down timer started around 15th October. No response from him on several emails.

The guys who visited the site in those days supposed to be a Halloween surprise.
I saw the etal svn up, and the same is old etal archive - atm most of it useless.

**Comment by mf022** [comment]
just an update [projectetal.com](https://www.projectetal.com/) become unreachable
https://downforeveryoneorjustme.com/projectetal.com
